tests/ludwig/encoders/test_text_encoders.py

```diff
@@ -107,7 +107,6 @@ def test_distil_bert(use_pretrained: bool, reduce_output: str, max_sequence_leng
     assert outputs["encoder_output"].shape[1:] == distil_bert_encoder.output_shape
 
 
-@slow
```

Comment:
Why did we remove the `slow` marker?

Reply:
@abidwael Want to make sure we start running tests for this text encoder. It's not actually that slow to run to be honest!